dietpi/dietpi-software

```diff
@@ -7984,8 +7988,26 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Installing
-			G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.noarmor.gpg -o /usr/share/keyrings/tailscale-archive-keyring.gpg
-			G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.tailscale-keyring.list -o /etc/apt/sources.list.d/tailscale.list
+			
+			G_AGUP
+			G_AGI gnupg
+			
+			if (( ${$G_DISTRO} == 5 )); then
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/buster.gpg -o /usr/share/keyrings/tailscale-archive-keyring.gpg
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/buster.list -o /etc/apt/sources.list.d/tailscale.list
+				G_EXEC apt-key add /usr/share/keyrings/tailscale-archive-keyring.gpg
+			fi
+
+			if (( ${$G_DISTRO} == 6 )); then
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.noarmor.gpg -o /usr/share/keyrings/tailscale-archive-keyring.gpg
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.tailscale-keyring.list -o /etc/apt/sources.list.d/tailscale.list
+			fi
+
+			if (( ${$G_DISTRO} == 7 )); then
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bookworm.noarmor.gpg -o /usr/share/keyrings/tailscale-archive-keyring.gpg
+				G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.tailscale-keyring.list -o /etc/apt/sources.list.d/tailscale.list
+			fi
+
```

Comment:
You could stick to the original version. Just replace the distro name with a variable like `$G_DISTRO_NAME`